services/auth/src/azure.js

```diff
@@ -150,7 +150,7 @@ builder.declare({
   route: '/azure/:account/containers',
   name: 'azureContainers',
   query: {
-    continuationToken: /^[A-Za-z][A-Za-z0-9]{2,62}$/,
+    continuationToken: /^[A-Za-z\/][A-Za-z0-9-\/]{2,62}$/,
```

Comment:
I've made the replacements but it looks like the test still fails and now some more are failing too. Are we maybe too strict in that definition?

Reply:
Hm, maybe continuation tokens for containers (and queues?) look different?  In `azure-entities` we define our own format (https://github.com/taskcluster/azure-entities/blob/7947f816a1b8b29264b9123b1c745b4623c07f7b/src/entity.js#L1064-L1093) but we don't do the same for blobs.

https://docs.microsoft.com/en-us/rest/api/storageservices/list-blobs only says that the value should be treated as opaque.  So, perhaps we should just be using `/.*/` for everything non-table-row-related..

Reply:
I've gone back to a simpler fix for this one case and filed [bug 1519277](https://bugzilla.mozilla.org/show_bug.cgi?id=1519277) for fixing the rest of these. I just want to get this landed sooner rather than later since it is causing all of our testruns to permafail.